action/kafkaProduce.py

```diff
@@ -16,11 +17,19 @@ def main(params):
 
         print "Created producer"
 
+        messageValue = params['value']
+        if 'base64DecodeValue' in params and params['base64DecodeValue'] == True:
+            messageValue = base64.b64decode(params['value'])
+
         # only use the key parameter if it is present
         if 'key' in params:
-            producer.send(params['topic'], bytes(params['value']), key=bytes(params['key']))
+            messageKey = params['key']
+            if 'base64DecodeKey' in params and params['base64DecodeKey'] == True:
+                messageKey = base64.b64decode(params['key'])
```

Comment:
Can just do `params['key'].decode('base64').strip()` instead of importing another library to do the decoding.